aea/cli/run.py

```diff
@@ -195,5 +197,17 @@ def _build_aea(
         )
         aea = builder.build(connection_ids=connection_ids, password=password)
         return aea
+    except AEANoDefaultLedgerException:
+        error_msg = "No default ledger specified. Please use 'aea config set agent.default_ledger LEDGER_ID'"
+        raise click.ClickException(error_msg)
+    except AEAWalletNoAddressException:
+        error_msg = (
+            "You haven't specified any private key for the AEA project.\n"
+            "Please add one by using the following commands:\n"
+            "\n"
+            f"    aea generate-key {DEFAULT_LEDGER}\n"
+            f"    aea add-key fetchai {PRIVATE_KEY_PATH_SCHEMA.format(DEFAULT_LEDGER)}\n"
+        )
+        raise click.ClickException(error_msg)
```

Comment:
we need custom exceptions as we need to catch this special cases.
We couldn't use just `ValueError` because these error messages wouldn't make sense in `AEABuilder` for, say, programmatic usage.